spec/features/address_bar_spec.rb

```diff
@@ -10,6 +10,9 @@ def query_string
   context 'when loading collection details page from search page' do
     before :all do
       visit '/search'
+      if page.has_link?('closeInitialTourModal')
+        find("#closeInitialTourModal").click
+      end
```

Comment:
The whole block of code can be extracted into a method and put it in helper file.